### PR TITLE
support multi event types

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: pkg/spoke/spokeagent.go [hunk 1]
```diff
@@ -92,10 +92,12 @@ func (o *WorkloadAgentOptions) RunWorkloadAgent(ctx context.Context, controllerC
 		return err
 	}
 
-	hubWorkClientHolder, err := workclient.NewWorkClientBuilder().
+	// TODO get source id
+	hubWorkClientHolder, err := workclient.NewWorkClientBuilder("hub/spoke-agent").
```
Comment (morvencao) [inline pkg/spoke/spokeagent.go:96]: why still have `hub` for agent source?

File: pkg/clients/workclient/builder.go [hunk 2]
```diff
@@ -13,12 +13,25 @@ import (
...
+		Group:    "io.open-cluster-management.works",
+		Version:  "v1alpha1",
+		Resource: "manifestbundle",
+	}
+	ManifestGVR = mqclient.GroupVersionResource{
+		Group:    "io.open-cluster-management.works",
+		Version:  "v1alpha1",
+		Resource: "manifest",
+	}
+)
```
Comment (morvencao) [inline pkg/clients/workclient/builder.go:33]: we will maintain this?  do we need "works" in group. 
I suppose this will be mapped to manifestwork GVR

**Comment by morvencao** [comment]
I'll merged this for now.

File: pkg/spoke/spokeagent.go (hunk 1 above)
Comment (skeeey) [reply pkg/spoke/spokeagent.go:96]: only agent id may be enough

**Pull request merged**